src/index.html

```diff
@@ -1,12 +1,51 @@
 <!DOCTYPE html>
 <html lang="en-US">
   <head>
-    <title>Webpack Starter Kit</title>
+    <title>Overlook</title>
   </head>
   <body>
-    <h1>Webpack Starter Kit</h1>
-
-    <img src="./images/turing-logo.png" alt="turing logo" height="100px" width="100px">
+    <header>
+      <h1>Overlook</h1>
+      <p>username</p>
+    </header>
+    <h2>Profile</h2>
+    <section class="profile">
+      <section class="spent-container">
+        <h3>Total Spent</h3>
+        <article id="totalSpent"></article>
+      </section>
+      <h3>Booked Rooms</h3>
+      <section class="booked-rooms-container">
+        <div>
+          <h4>Room Number
+            <article class="room-number" id="roomNumber"></article>
+          </h4>
+        </div>
+        <div>
+          <h4 class="date-of-stay">Date of Stay
+            <article id="dateStayed"></article>
+          </h4>
+        </div>
+      </section>
+    </section>
+    <h2>Book A Room</h2>
+    <section>
+      <div class="book-room-container">
+        <label for="date">Select A Date</label>
+        <input type="date" id="date" class="date" name="book-date">
+      </div>
+      <div class="book-room-container">
+        <label for="roomType">Select Room Type</label>
+        <select id = "roomType" class="roomType">
+          <option value="default">Select Room Type</option>
+          <option value="single room">Single Room</option>
+          <option value="residential suite">Residential Suite</option>
+          <option value="junior suite">Junior Suite</option>
+          <option value="suite">Suite</option>
+        </select>
+      </div>
```

Comment:
Fair amount of divs here. May want to consider switching up to something more semantic